Frontend/ui-driver/src/Screens/UploadDrivingLicenseScreen/View.purs

```diff
@@ -53,9 +75,16 @@ screen initialState =
   , globalEvents : [(\push -> do
     _ <- JB.storeCallBackImageUpload push CallBackImageUpload
     _ <- runEffectFn1 consumeBP unit
+    if initialState.props.successfulValidation then do
+      _ <- launchAff $ flowRunner defaultGlobalState $ redirectScreen push RedirectScreen
+      pure unit
+    else pure unit
+    void $ launchAff $ EHC.flowRunner defaultGlobalState $ runExceptT $ runBackT $
+              if (initialState.props.validateProfilePicturePopUp == true)  then  lift $ lift $ doAff do liftEffect $ push $ AfterRender  else pure unit 
```

Comment:
`initialState.props.validateProfilePicturePopUp == true` is `initialState.props.validateProfilePicturePopUp` only